typescript-intro/typescript-intro-quiz-notes.md

```diff
@@ -0,0 +1,44 @@
+# typescript-intro-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- When was TypeScript introduced?
+  It was created in 2012
+- Who created TypeScript?
+  Microsoft
+- What does the TypeScript `Compiler` do?
+
```

Comment:
Just missing one quiz question here! If you're not completely sure, then use your best guess. I'm not looking for the "correct" answer, I'm looking for at least an attempt!